topics/assembly/tutorials/assembly-with-preprocessing/tutorial.md

```diff
@@ -0,0 +1,629 @@
+>        - *"Type"*: `fastqsanger.gz`
+>        - *"Name"*: `Illumnia PE data` (or similar)
+>        - *"Add nametag for name:"* {% icon param-check %}
+>
+>          > ### {% icon comment %} Name tags in the analysis
+>          > We are going to treat the Illumina- and the Nanopore-sequenced data
+>          > separately in this tutorial up to the actual genome assembly step.
+>          >
+>          > To make it easier to keep track of the two branches of the analysis, we
+>          > recommend the use of Galaxy's dataset **name tags**.
+>          > A name tag will automatically propagate to any new dataset derived
+>          > from the tagged dataset.
+>          {: .comment}
+>
+>          Checking this option tells Galaxy to reuse the collection name above
+>          as a name tag on the collection.
+>
+>        - Click **Upload**
+>
+> 3. Import Nanopore-sequenced reads data from [Zenodo](https://zenodo.org/record/3732359)
+>
+>    ```
+>    https://zenodo.org/record/3732359/files/SRR10902284_ONT.fq.gz
+>    https://zenodo.org/record/3732359/files/SRR10948474_ONT.fq.gz
+>    https://zenodo.org/record/3732359/files/SRR10948550_ONT.fq.gz
+>    ```
+>
+>    Again, we want to upload this data to our history in structured form.
+>    To do so:
+>
+>      - Copy the above list of links
+>      - Open the Galaxy Upload Manager
+>        ({% icon galaxy-upload %} on the top right of the tool panel)
+>      - In the pop-up window, switch to the **Rule-based** tab and select
+>        - *"Upload data as:"*: `Collection(s)`
+>        - *"Load tabular data from:"*: `Pasted Table`
+>        - Paste the copied links into the text field
+>        - Click **Build**
+>      - In the next screen, select
+>        - {% icon param-repeat %} *"Column"*: `Using a Regular expression`
+>          - *"From Column"*: `A`
+>          - {% icon param-check %} *"Create columns matching expression groups."*
+>          - *"Regular Expression"*: `.+/(SRR\d+)_ONT.fq.gz`
+>          - *"Number of Groups"*: `1`
+>          - Click **Apply**
+>        - {% icon param-repeat %} *"Rules"*: `Add / Modify Column Definitions`
+>          - {% icon param-repeat %} *"Add Definition"*: `URL`
+>            - *"URL"*: `A`
+>          - {% icon param-repeat %} *"Add Definition"*: `List Identifier(s)`
+>            - *"List Identifier(s)"*: `B`
+>          - Click **Apply**
+>        - *"Type"*: `fastqsanger.gz`
+>        - *"Name"*: `Nanopore data` (or similar)
+>        - *"Add nametag for name:"* {% icon param-check %}    
+>        - Click **Upload**
+>
+{: .hands_on}
+
+
+# Read trimming and quality control
+
+1. Trimming of Illumina reads with fastp / visualizing results with MultiQC
+
+   **TO DO**
+
+2. QC of Nanopore reads with NanoPlot and/or FastQC
+
+   **TO DO**
+
+# Subtraction of reads mapping to the human reference genome
+
+## Map Illumina reads with bowtie2
+
+In this tutorial, we are using **Bowtie2** for mapping our short-reads data to
+the human genome. *BWA
```

Comment:
Meant to mention *bwa-mem* as the obvious alternative here.
Could also be worthwhile to discuss RNA-Aligners? Since what must have been isolated initially from the BALF is RNA?
I did some experiments with STAR: results are very similar to bowtie2/bwa ones. So not worth decribing it in detail, but maybe mention it?